js/settingsHelpers.js

```diff
@@ -650,6 +695,27 @@ export const unlinkFirebaseAuthProvider = async (providerType, userData) => {
   }
 };
 
+export const updateToNoReplyEmail = async (uid, noReplyEmail) => {
+    document.getElementById('loginUpdateFail').style.display = 'none';
+    document.getElementById('loginUpdateSuccess').style.display = 'none';
+    
+    const valuesForFirebaseAuth = {};
+    valuesForFirebaseAuth['email'] = noReplyEmail;
+    valuesForFirebaseAuth['uid'] = uid;
+    
+    try {
+      await updateFirebaseAuthEmail(noReplyEmail);
+      return true;
+    } catch (error) {
+      console.error(`Error: updateFirebaseAuthEmail(): ${error}`);
+      throw error;
+    }
+};
+
```

Comment:
line 712 - 718 => Are these lines that we do it on purpose?

Reply:
Thanks - removed extra spacing.